www/src/py_dom.js

```diff
@@ -342,6 +342,7 @@ $StyleDict.__setattr__ = function(self,attr,value){
         self.js.cssFloat = value
         self.js.styleFloat = value
     }else{
+        attr = attr.replace('_', '-')
         switch(attr) {
```

Comment:
Actually, when '-' are supported this way the next switch should be rewritten to also support, e.g., `min-height`, `max-height` etc.